src/mod/applications/mod_dptools/mod_dptools.c

```diff
@@ -953,6 +955,14 @@ SWITCH_STANDARD_APP(eavesdrop_function)
 		if (switch_true(bridge_bleg)) {
 			flags |= ED_BRIDGE_WRITE;
 		}
+		if (switch_true(bug_top)) {
+			flags &= ED_BUG_BOTTOM;
```

Comment:
shouldn't this read `~ED_BUG_BOTTOM`? if the intention is to reset the bit?

Reply:
yup, it was not added. I have already push a fix for that

Reply:
i think that `~` is still missing here...

Reply:
OH, I miss those. Updated it